### ISSUE TITLE
every time when I write non-english word and press Cmd-S

### ISSUE THREAD

**Issue by vhor** [careteditor/caret#619]
The following error has occurred:
```
{
    "message": "Cannot read property 'end' of undefined",
    "version": "4.0.0-beta6",
    "os": "macOS High Sierra",
    "path": "extensions/Editor/Editor.Spellcheck/Editor.Spellcheck.js",
    "code": "",
    "line": 5,
    "column": 3551,
    "stack": [
        "at _showCorrectionsTimeoutId.setTimeout (extensions/Editor/Editor.Spellcheck/Editor.Spellcheck.js:5:3551)"
    ]
}
```

**Comment by astoilkov** [comment]
Hi @vhor,

We have fixed the problem in our internal build. I will write in this thread when we release the next version.

Thanks for reporting.

**Comment by astoilkov** [comment]
The fix now landed in the latest version of Caret 4.0 Beta.

Closing this issue. Thanks for reporting!
